higher_health/templates/healthcheck_questionnaire.html

```diff
@@ -274,12 +285,11 @@ <h3 class="heading-subtitle heading-subtitle--edu-label">Confirmation</h3>
     }
   };
 
-
-  let universities = JSON.parse(document.getElementById('universities-data').textContent),
-  campuses = JSON.parse(document.getElementById('campus-data').textContent);
     let prov = document.getElementById('id_facility_destination_province'),
       uni = document.getElementById('id_facility_destination_university'),
+      uni_other = document.getElementById('id_facility_destination_university_other'),
```

Comment:
I find the `uni_other` and `other_university` variables too close in naming, and so very confusing.
I think we should rename the `uni_other` to something more descriptive of what it is.